app/src/main/java/com/geeksville/mesh/NodeInfo.kt

```diff
@@ -29,10 +30,16 @@ data class MeshUser(
     val shortName: String,
     val hwModel: MeshProtos.HardwareModel,
     val isLicensed: Boolean = false,
+    val role: DeviceConfig.Role
```

Comment:
using protobuf enums for types (like hwModel) is problematic, instead use `Int` to work with the "_VALUE" suffix (e.g., CLIENT -> CLIENT_VALUE) and role.number

then you can tag "= 0" as default value (CLIENT) so we don't need any extra handling elsewhere like in testing

```kotlin
data class MeshUser(
    val id: String,
    val longName: String,
    val shortName: String,
    val hwModel: MeshProtos.HardwareModel,
    val isLicensed: Boolean = false,
    val role: Int = 0, // ConfigProtos.Config.DeviceConfig.Role.CLIENT_VALUE
) : Parcelable {
```